### PR TITLE
Update the version of  packages

### PR DESCRIPTION
**Pull request by ZhaodongTian**
Update the version of packages

### CODE REVIEW

File: src/dotnet-svcutil/lib/src/DebugLogger.cs [hunk 1]
```diff
@@ -101,10 +125,31 @@ void NuGet.Common.ILogger.LogInformationSummary(string data)
             LogMessage(data, LogTag.LogMessage);
         }
 
+        void NuGet.Common.ILogger.Log(NuGet.Common.ILogMessage message)
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/src/DebugLogger.cs:128]: Why is this change needed for updating the Newtonsoft packages?

Comment (ZhaodongTian) [reply src/dotnet-svcutil/lib/src/DebugLogger.cs:128]: > Why is this change needed for updating the Newtonsoft packages?

The interface ILogger in Higher version of NuGet.Common package has defined more methods, we should  have the implementation in the inherited class

**Review by HongGit — APPROVED** [review]
